lib/mongoose.js

```diff
@@ -35,7 +35,7 @@ module.exports = function FilterablePlugin(Filterable, schema, options) {
             return [
                 field,
                 {
-                    type: _.contains(VALID_TYPES, schemaField.type)? schemaField.type : null,
+                    type: _.contains(VALID_TYPES, schemaField.instance)? schemaField.type : null,
```

Comment:
https://github.com/Automattic/mongoose/issues/1430#issuecomment-16387541

We should use `schenaField.type`